src/telemetry/graph.jsx

```diff
@@ -60,16 +64,25 @@ export default class TelemetryContainer extends React.Component {
 
   render() {
     const { id, title } = this.props;
+
     return (
-      <div id={id} key={id} className='criteria-widget'>
+      <div id={id} key={id} className="criteria-widget">
         <header>
-          <h3 className='graph-title'>
-            <a className='graph-title-link' ref={a => this.graphTitleLink = a}>{title}</a>
+          <h3 className="graph-title">
+            <a
+              className="graph-title-link"
+              ref={a => (this.graphTitleLink = a)}>
```

Comment:
/* eslint-disable jsx-a11y/anchor-is-valid */

> Why is this rule disabled?

It has `ref` attribute in the anchor tag which makes it a clickable link. 
Linting error appears if I don't make it `href`, and if I make it an `href` attr react gives an error and breaks code. 

`ref` concept used here looks like this
https://reactjs.org/docs/refs-and-the-dom.html 

while linting error is saying that anchor isn't valid
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md

I am confused about solving it... how can I do it?

Reply:
@sarah-clements @djmitche your thoughts are required.

Reply:
Let's handle this in another PR.  Here's what I think is happening (and don't be confused by the similar spelling of `ref` and `href` - eslint is not confusing the two).

The link being defined here has an href which is only known after a fetch is complete (defined earlier in the same file).  The solution was to use the `ref` attribute, which is a React thing that allows you to get a reference to a DOM object, to put a reference to the link object into `this.graphTitleLink`.  Then, when the fetch is complete, there's a call to `this.graphTitleLink.setAttribute('href', telemetryUrl)` to set the href.

But eslint is complaining that this is an `a` element without an `href` -- it can't see that we plan to fill in the `href` attribute later.

I suspect the easy fix to this (and, again, let's do this in another PR!) is to put a "fake" `href` into the `a` element to start with, so that the `setAttribute` call just replaces the fake value with the real value.  I think `#` is a common "fake" href value.  So, `<a href="#" className="graph-title-link" ref={a => (this.graphTitleLink = a)}>`.

Reply:
@djmitche I tried doing this but it didn't work. Still gives the error. 
I tried 
href=""
href="#"
href={}
nothing worked!

It was also mentioned somewhere on a GitHub issue. It didn't work so I disabled it.

Reply:
Sounds good.  We can revisit in a dedicated PR.